src/plugins/ovp/screens/video/index.jsx

```diff
@@ -29,6 +29,7 @@ class VideoScreen extends AbstractNodeScreen {
       this.renderPreviewButtons(),
       this.renderNodeStatus(),
       this.renderPublishForm(),
+      this.renderLabels(),
```

Comment:
@rsumilang this should be LabelsForm or (since it's both displaying and allowing the user to change them)